.github/workflows/tests.yml

```diff
@@ -17,6 +17,7 @@ jobs:
       skip-arm-tests: ${{ steps.check-arm-labels.outputs.result }}
     env:
       HOMEBREW_FORCE_HOMEBREW_ON_LINUX: 1
+      HOMEBREW_SIMULATE_MACOS_ON_LINUX: 1
```

Comment:
Not blocking but: may want to consolidate these: https://github.com/Homebrew/brew/pull/10767#discussion_r588237683